trpcultivate_phenotypes/tests/src/Kernel/Display/DisplayValidationResultWindowTest.php

```diff
@@ -0,0 +1,406 @@
+<?php
+
+namespace Drupal\Tests\trpcultivate_phenotypes\Kernel\Display\DisplayValidationResultWindowTest;
+
+use Drupal\Tests\tripal_chado\Kernel\ChadoTestKernelBase;
+
+/**
+ * Tests Tripal Cultivate Phenotypes Validation Result Window.
+ *
+ * @group trpcultivate_phenotypes
+ * @group displays
+ */
+class DisplayValidationResultWindowTest extends ChadoTestKernelBase {
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  protected static $modules = [
+    'tripal',
+    'tripal_chado',
+    'trpcultivate_phenotypes',
+  ];
+
+  /**
+   * Drupal render service.
+   *
+   * @var Drupal\Core\Render\RendererInterface
+   */
+  protected $renderer;
+
+  /**
+   * {@inheritDoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    // Set test environment.
+    \Drupal::state()->set('is_a_test_environment', TRUE);
+
+    $this->renderer = \Drupal::service('renderer');
```

Comment:
Since this is a service call, you should be able to just do the following:
```suggestion
    $this->renderer = $this->container->get('renderer');
```